### DISCUSSION TITLE
Way to remove tracker

### DISCUSSION THREAD

**Discussion by Ygarr** [Hazuki0x0/YuzuBrowser#200] [Q&A] [answered]
Is there way to build without 3 trackers wich is come along with this browser?

**Comment by hazuki0x0** [comment, marked as answer]
You can build without Firebase, just reverse the steps below for `app/build.gradle` and top level of `build.gradle`.
https://firebase.google.com/docs/crashlytics/get-started?platform=android
https://firebase.google.com/docs/android/setup

And remove the following code in `app/build.gradle`
```
android.applicationVariants.all { variant ->
    def googleTask = tasks.findByName("process${variant.name.capitalize()}GoogleServices")
    googleTask.enabled = "debug" != variant.name
}
```
